content/sensu-core/2.0/getting-started/recommended-hardware.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Hardware Requirements and Recommendations"
+linkTitle: "Hardware Requirements"
+description: "Guidelines for what you'll need for deploying Sensu Core to production."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+platforms: ["Ubuntu/Debian", "RHEL/CentOS", "Windows"]
```

Comment:
We don't actually use the platform content on this page so `platformContent` should probably be set to `false` and we can remove this line!